INSTALL.rst

```diff
@@ -244,6 +244,16 @@ The commands for ``OS X`` are the same as in ``Linux``.
     For developers, honcho is recommended and will make your life
     easier because it launches all the servers together as it finds the ``Procfile``.
 
+.. note::
+
+    In case you got an error ``'ffi.h' file not found`` while installing invenio,
+    try to intall the ``libffi`` library e.g. via ``brew``:
+
+    .. code-block:: console
+
+        $ brew install libffi
```

Comment:
Better include it together with https://github.com/inveniosoftware/invenio/pull/3109/files#diff-26dee18463b5da190f343abcc74cf708R214. Thanks